packages/sirius-web/backend/sirius-web-tests/pom.xml

```diff
@@ -90,6 +90,11 @@
 			<artifactId>sirius-components-portals-tests</artifactId>
 			<version>2025.4.0</version>
 		</dependency>
+		<dependency>
```

Comment:
The dependency is needed by the runner (for the input).
We could have a dedicated test module for the view-fork component (we have one for diagrams for example). I don't know if this is something we want or if it is overkill. So I went for the simpler solution, let me know if you prefer to move the class somewhere else.

Reply:
:+1: